src/citrine/informatics/predictors.py

```diff
@@ -83,8 +83,13 @@ class SimpleMLPredictor(Serializable['SimplePredictor'], Predictor):
         Descriptors that represent outputs of relations
     latent_variables: list[Descriptor]
         Descriptors that are predicted from inputs and used when predicting the outputs
-    training_data: DataSource
-        Source of the training data, which can be either a CSV or an Ara table
+    training_data: Optional[List[DataSource]]
+        Sources of training data. Each can be either a CSV or an Ara table.
+        Candidates from multiple data sources will be combined into a flattened list and deduplicated by uid
```

Comment:
Should some or all of this description be included in the user guide as well?

Reply:
That's a good idea. I added a training data section to the user docs.